recipes/cwinpy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "cwinpy" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3caedf3aac6b8ef9bc0df8467e1610587c1fae3b2a0266ee6c839d45a79e5cb0
+
+build:
+  entry_points:
+    - cwinpy_knope = cwinpy.knope.knope:knope_cli
+    - cwinpy_knope_dag = cwinpy.knope.knope:knope_dag_cli
+    - cwinpy_knope_generate_pp_plots = cwinpy.knope.testing:generate_pp_plots
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - astropy
+    - bilby >=0.5.8
+    - bilby_pipe >=0.2.0
+    - gwpy
+    - lintegrate
+    - matplotlib
+    - numba
+    - numpy >=1.15
+    - pycondor
+    - python >=3.6
+    - python-lal
```

Comment:
`python-lal` and `python-lalpulsar` are not available for Windows, so I don't think this recipe can be `noarch`. I would suggest to either update the feedstocks if possible, or skip Windows.

Reply:
@npavlovikj, I have submitted a number of different recipes that are `noarch: python` but rely upon dependencies that are not available for one or more platforms. I thought this was a fairly standard situation, given the benefits of `noarch: python`.

Reply:
As long as only the dependencies are missing but the actual package is independent of the host system, this can be merged. The dependencies must be available on Linux as we build the recipe there, failing jobs on Windows here in staged-recipes are ok.

Reply:
I believe all of these requirements are met.

Reply:
Thank you for the clarification @xhochy !